congress/metadata/C001111.yaml

```diff
@@ -0,0 +1,2 @@
+name: LWVSPA
+link: https://lwvspa.org/congressional-representatives-pinellas-county/
```

Comment:
I can't figure out where Crist's image ultimately comes from here, so it's not safe to assume it's free of copyright. Options:

* http://crist.house.gov/images/crist-head.jpg
* https://twitter.com/repcharliecrist avatar (seems like the same)